wp-content/plugins/cf-demo-lessons/cf-demo-lessons.php

```diff
@@ -27,7 +30,12 @@ public function __construct()
 			add_action('init', array($this, 'lessonRegisterCustomTaxonomy'));
 			add_action('cmb2_admin_init', array($this, 'registerLessonMetabox'));
 		}
-
+		
+		/**
+		 * lessonRegisterPostType
```

Comment:
The documentation for functions and methods should describe what the function/method *does*. See https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods (only the Description, @param and @return are needed)